nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/FetchDistributedMapCache.java

```diff
@@ -151,6 +164,35 @@ public FetchDistributedMapCache() {
     }
 
     @Override
+    protected Collection<ValidationResult> customValidate(ValidationContext validationContext) {
+        List<ValidationResult> results = new ArrayList<>(super.customValidate(validationContext));
+
+        PropertyValue cacheEntryIdentifier = validationContext.getProperty(PROP_CACHE_ENTRY_IDENTIFIER);
+        boolean elPresent = false;
+        try {
```

Comment:
Hoping to remove this try/catch if [NIFI-4590](https://issues.apache.org/jira/browse/NIFI-4590) is implemented first.